hv-rhel7.x/hv/hyperv_bl_drv.conf

```diff
@@ -0,0 +1,3 @@
+# Do not load the 'mlx ib' drivers
```

Comment:
I don't see a need for creating a separate file in repository.
Just using two lines echo from install script will suffice.

Reply:
Its better to have static file for better visibility.